src/erl_cache.app.src

```diff
@@ -1,13 +1,15 @@
 {application, erl_cache,
  [
-  {description, ""},
-  {vsn, git},
-  {registered, []},
+  {description, "A generic in-node caching library with memoization support"},
+  {vsn, "1.7.1-alertlogic"},
```

Comment:
IDK if this is the best to do or if it's preferable to support the "git" thing still.

Reply:
`rebar3 hex publish` supports git tags as longs as they are annotated. this is unnecessary.